experiment/build/gcb_build.py

```diff
@@ -84,7 +84,9 @@ def _build(
 
         # Use "s" suffix to denote seconds.
         timeout_arg = '--timeout=%ds' % timeout_seconds
-        worker_pool_arg = f'--worker-pool={WORKER_POOL_NAME}'
+        worker_pool_name = yaml_utils.read(experiment_config_path()).get(
+            'pool_name', DEFAULT_WORKER_POOL_NAME)
```

Comment:
I guess we can get `pool_name` in this way during experiments, but not in unit tests?
IIUC, unit tests do not copy the `experiment-config.yaml` to this directory like in experiments, hence CI fails on this line because it cannot find that file.
An alternative solution that will work for both is to carry that information all the way here. I am working on that now, but please let me know if there is a better solution.

Reply:
I didn't design config very well in fuzzbench. some config is put in the env other is read into memory and passed around. I think an env var is good for something like this, definitely don't read that file again please.